src/main/java/org/apache/commons/collections4/bloomfilters/FilterConfig.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.collections4.bloomfilters;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Filter configuration class.
+ * <p>
+ * This class contains the values for the filter configuration and is used to
+ * convert a ProtoBloomFilter into a BloomFilter.
+ * </p>
+ *
+ * @see <a href="http://hur.st/bloomfilter?n=3&p=1.0E-5">Bloom Filter
+ *      calculator</a>
+ *
+ * @since 4.5
+ */
+public final class FilterConfig implements Serializable {
```

Comment:
Seems like an odd class name, why not prefix the class name with 'Bloom'? Would this class _ever_ apply to non-bloom classes?